secret-contracts/calculator-with-permits/src/msg.rs

```diff
@@ -34,8 +34,8 @@ pub enum QueryMsg {
 #[serde(rename_all = "snake_case")]
 pub enum QueryWithPermit {
     CalculationHistory {
-        page: Option<u32>,
-        page_size: u32,
+        page: Option<Uint128>,
+        page_size: Uint128,
     },
 }
```

Comment:
The paging parameters actually don't have to be uing128 because the underlying types don't support such large amounts of data

Reply:
that's only because it is defined that way by the exercises requirements